src/components/TransactionPreview/ActionPreview.tsx

```diff
@@ -432,14 +458,35 @@ const ActionPreview = ({
   editButtonDisabled = false,
   showSignButton = false,
   showEditButton = false,
+  isSubmitTrue = false,
   setIsTransactionDone,
   showStatus = true,
   showGasAssetSelect = false,
 }: TransactionPreviewInterface) => {
+  const [minutes, setMinutes] = useState(0);
+  const [seconds, setSeconds] = useState(30);
```

Comment:
you can use just `seconds` and use `moment` lib to show, please fix this and rest of related code